localstack/testing/snapshots/transformer_utility.py

```diff
@@ -165,18 +165,36 @@ def sqs_api():
         """
         return [
             TransformerUtility.key_value("ReceiptHandle"),
-            TransformerUtility.key_value("SenderId"),
+            # account ID
+            TransformerUtility.key_value(
+                "SenderId", value_replacement="111111111111", reference_replacement=False
+            ),
             TransformerUtility.jsonpath("$..MessageAttributes.RequestID.StringValue", "request-id"),
             KeyValueBasedTransformer(_resource_name_transformer, "resource"),
-            KeyValueBasedTransformer(_signing_cert_url_token_transformer, replacement="token"),
+        ]
+
+    @staticmethod
+    def sns_api():
+        """
+        :return: array with Transformers, for sns api.
+        """
+        return [
+            TransformerUtility.key_value("SequenceNumber"),  # this might need to be in SQS
+            TransformerUtility.key_value(
+                "Signature", value_replacement="<signature>", reference_replacement=False
+            ),
+            TransformerUtility.key_value("MD5OfBody", "<md5-hash>", reference_replacement=False),
```

Comment:
should we not maybe leave this in per default? or are you having issues with bodies containing data that is dynamically generated by the tests? (like passing ARNs through a message body).

Reply:
+1 
If the md5 hash matches in most cases, we should not always exclude it. Is this the case for SNS?
Ideally we would only exclude md5-hashes for individual test cases with a note why it has to be replaced (e.g. includes timestamp).

Reply:
The MD5 hash almost never matches: the body of the SNS message always contains a timestamp.
It only matches when `RawDelivery` is set to True, as the body is the string message passed. I had to manually add the MD5 transformer to almost every test. I thought the MD5 hash could just be checked with a unit test.
Otherwise, I had an over engineered way to do, which was to delete the transformer from the `sns_api`.
What do you think we should do in this case?

Reply:
alright :) i think in that case, having unittests should be fine. 
alternatively we could also create another api-helper, which excludes the md5-transformer and use this one for the `RawDelivery` messages

Reply:
Yes, it could work as well. The issue also that I didn't think about is that in some tests, we have both `RawDelivery` and regular in the same test. So I have to ignore the hash. Maybe I will manually check the MD5 of raw messages in the integration tests? As it's AWS validated, it will check the behaviour when we regenerate the snapshots

Reply:
I see :) yes, sounds good. but I think it would also be fine to cover the md5-checksum in unittests as you suggested before.

Reply:
The md5 is part of SQS, and is covered inside the integration tests there, there shouldn't be any regression on that side.